### ISSUE TITLE
Meepo innate free  clone bug

### ISSUE THREAD

**Issue by onic2999** [LegendsOfDota/LegendsOfDota#399]
LoD specifies that meepo gets a free clone with aghs without divided we stand,well i tested it and it didn't worked i just increases my stats not gives free clone,selling it,dying didn't solve anything.

**Comment by ash47** [comment]
Must have been patched recently. P

**Comment by XAgent1990** [comment]
Yup it's true, this was patched long ago and isn't the case anymore. A missleading LoD Fact :D

**Comment by ash47** [comment]
Cool, will get this fixed up at some point
